src/landscape/C4Texture.cpp

```diff
@@ -47,7 +47,7 @@ C4Texture::~C4Texture()
 }
 
 C4TexMapEntry::C4TexMapEntry()
-		: iMaterialIndex(MNone), pMaterial(nullptr)
+		: iMaterialIndex(MNone)
```

Comment:
`MNone` is a constant iirc, so could be moved to the declaration (unless of course its declaration isn't yet available in C4Texture.h)